src/socket/chat.ts

```diff
@@ -123,7 +121,6 @@ export const joinPasswdChatRoom = (
   navigate: NavigateFunction,
   setState: Dispatch<SetStateAction<string>>,
   setClose: () => void,
-  setRoom: Dispatch<SetStateAction<number | undefined>>,
 ): void => {
   const socket = getSocket();
```

Comment:
```suggestion
  const socket = getSocket();
  const navigate = useNavigate();
```

이 함수 내부에서 생성해서 써도 됩니다!

Reply:
컴포넌트에서만 훅을 쓰는게 가능한줄 알았는데 ?? 몰랐네요

Reply:
음???????안되면 말해주세요....